### ISSUE TITLE
Windows: Include repositories from WSL 

### ISSUE THREAD

**Issue by MrLuje** [mohitsinghs/vscode-repo#2]
I am using vscode on windows 10 but mainly using WSL2 so 90% of my repositories are in WSL.
By default, the extension shows windows' repository but not the WSL ones, but is would be nice to display both windows & WSL repositories.

Workaround :
* go to %userprofile%\\.vscode\extensions\mohitsingh.repo-0.2.2-win32-x64\bin
* run `repo.exe init`
* edit `%APPDATA%\repo.yml` and add the following
```yaml
roots: 
  # host paths
  - path: c:\users\xxxxx  # put your home directory here
  # wsl paths
  - path: \\wsl$\YOUR_WSL_DISTRIB_NAME\home\YOUR_USERNAME\
```
* now it should properly show both windows & wsl repositories

**Comment by mohitsinghs** [comment]
Hi @MrLuje, Thanks for the details. I don't use Windows, but I definitely want to improve Windows support. I think we should add this workaround to README as of now. ( Maybe a PR ? )  Meanwhile, I'll be adding ability to detect and scan WSL home in the binary.

I am also in process of moving some calculations from extension to binary and making paths configurable through extension itself.

**Comment by MrLuje** [comment]
Sure, I'll have a PR for the Readme by the weekend

**Comment by MrLuje** [comment]
In fact, the workaround is not fully working.
It properly builds the tree view, but fails to open repositories on click

![image](https://user-images.githubusercontent.com/632075/183375252-cfe33d77-fe96-4486-989f-ed565ee6cdf9.png)

**Comment by mohitsinghs** [comment]
Those seems to be valid wsl paths.
I guess, I'll have to arrange a Windows Machine to debug and verify this. Hopefully, It will be fixed then. I don't have any ETA as of now but I'm on it.

**Update**
I just quickly tested it on a windows computer and your workaround works great, so maybe something else is going on there. I'll look further into it once I merge tree building for our binary.

**Comment by MrLuje** [comment]
Looks like the first `\` disappeared 
The working path if `\\wsl$\Ubuntu\home\vince\src\github\rancher\charts` (starting with `\\`)

**Comment by mohitsinghs** [comment]
I just published v0.3.0 of extension that supports tree view on Windows and is slightly faster. 

Unfortunately, WSL support is still unsolved because scanning WSL from Windows is really slow for some reasons. I've added notes in README for limited support of WSL.

**Comment by mohitsinghs** [comment]
Closing this in favor of #4 where we'll be tracking support for scanning remote environments including WSL.
